### ISSUE TITLE
Link to navigate using ABRP

### ISSUE THREAD

**Issue by johan12345** [johan12345/EVMap#22]
Similar to the integration with Chargeprice.app (#12), we could also add a button to navigate to a selected charger using A Better Routeplanner (ABRP).

This also requires a change on ABRP's side, see here:
https://forum.abetterrouteplanner.com/topic/1694-direct-link-to-navigate-to-given-location/

**Comment by techmaved** [comment]
So I would like to shine a little bit light at this topic. It seems like that your suggestion has made it as a feature into ABRP. As an example I created [this route](https://abetterrouteplanner.com/?plan_uuid=fd5eba7f-0d7f-4d71-b9ba-956e59c03c00) from Berlin to Munich. You can create the link easily by planning a route and then clicking on the _Share_ icon and by selecting _Share ABRP link_.

**Comment by johan12345** [comment]
That is a link to a fully planned *route* (using a randomly generated ID), not to a *destination* (using its coordinates or address).

**Comment by 360path** [comment]
I know it would be a huge undertaking, but it would be great if there was some route planning ability in EVMap, because ABRP is absolutely clunky and buggy in use (especially the AAOS version).

Anyway, the navigate button should ideally ask which Navigation app should be used, if there are several installed (OsmAnd, CoMaps, Organic Maps, ABRP, Google Maps, ...).

**Comment by 360path** [comment]
> Anyway, the navigate button should ideally ask which Navigation app should be used, if there are several installed (OsmAnd, CoMaps, Organic Maps, ABRP, Google Maps, ...).

I just found out that tapping the location entry in the selected charger details opens a menu which asks which Navigation app should be used (I can select ABRP, CoMaps, Google Maps and a bunch of public transport and outdoor sports apps). 😊

**Comment by johan12345** [comment]
> I just found out that tapping the location entry in the selected charger details opens a menu which asks which Navigation app should be used

Yes, for the navigation button that is also the case, at least if you disable the option "start navigation immediately" in the settings (which only works with Google Maps).
